x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -166,6 +166,27 @@ eth0           |gamma          |fe80::cae2:65ff:fece:feb9
 eth0           |epsilon        |[fe80::cae2:65ff:fece:feb9, fe80::cae2:65ff:fece:fec0, fe80::cae2:65ff:fece:fec1]|fe80::cae2:65ff:fece:fec1|fe80::cae2:65ff:fece:fec1
 ;
 
+
+inWithWarningsRegex#[skip:-8.13.99, reason:regex warnings in tests introduced in v 8.14.0]
```

Comment:
This is just a duplicate of the previous test. 
At first I changed it in place, but it breaks BWC, and I don't like the idea of disabling a test for previous versions just because of this. At the same time, I like the idea of having at least one test that proves that regex warnings work.

Reply:
Checking 👀

Reply:
IMO if those work, we can drop this duplicate, which is just checking that the mechanism works. But can also stay.

Reply:
I had a look at the two tests, and I think they will work fine with this change.
I'll re-enable them with a separate PR after this one is merged, so that if something is not completely covered and we see further random failures, we can easily revert only that part.